README.md

```diff
@@ -20,8 +20,19 @@ This package uses [composer](https://getcomposer.org) so you can just add `trade
 ## Usage
 
 ### Chef API Wrapper
-Once you've created a chef API client using [jenssegers/php-chef], you can instantiate the wrapper and perform actions.  For example:
 ```php
-$chef = new TraderInteractive\Chef\Chef($chefApi);
-$chef->patchDatabag('data', 'item', ['url' => 'http://example.com']);
```

Comment:
It might be useful to have an "Upgrading from previous versions" section with this old usage and how to upgrade to the new usage.